### PR TITLE
Fix failing beta test for ARM shoot 

### PR DESCRIPTION
**Pull request by acumino**
**How to categorize this PR?**

/area testing
/kind bug

**What this PR does / why we need it**:
This PR fixes failing beta `testmachinery` test for ARM shoot. 

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```other operator
NONE
```

### CODE REVIEW

File: test/testmachinery/shoots/vpntunnel/vpntunnel.go [hunk 1]
```diff
@@ -162,6 +162,8 @@ var _ = ginkgo.Describe("Shoot vpn tunnel testing", func() {
 			"AppLabel":    copyLabel,
 			"SizeInMB":    500,
 			"KubeVersion": f.Shoot.Spec.Kubernetes.Version,
+			// Here it is considered that all worker pool of shoot have same architecture.
```
Comment (rfranzke) [inline test/testmachinery/shoots/vpntunnel/vpntunnel.go]:
```suggestion
			// Here it is assumed that all worker pool of shoot have same architecture.
```

**Review by rfranzke — COMMENTED** [review]
There are quite some more hard-coded images in the `test/framework/resources/templates` directory. What about them?

File: test/testmachinery/shoots/vpntunnel/vpntunnel.go (hunk 1 above)
Comment (acumino) [reply test/testmachinery/shoots/vpntunnel/vpntunnel.go]: Everything else is multi-arch in that except https://github.com/gardener/gardener/blob/9bd3ee31d8e5e8cd0dc4c37e77dec09e6218ea50/test/framework/resources/templates/logger-app.yaml.tpl#L19-L23 which can't be done cause I am not able to find any other image which gives the same functionality.
Also, the respective image runs in shoot controlplane so test will not be affected by it.

Comment (rfranzke) [reply test/testmachinery/shoots/vpntunnel/vpntunnel.go]: OK, thanks!

**Review by rfranzke — COMMENTED** [review]
/lgtm
/approve
/milestone v1.55

**Pull request merged**